trace-dispatcher/trace-dispatcher.cabal

```diff
@@ -150,3 +157,25 @@ test-suite trace-dispatcher-test
                        -Wredundant-constraints
                        -Wmissing-export-lists
                        -Wno-incomplete-patterns
+
+benchmark trace-dispatcher-bench
+  type:                exitcode-stdio-1.0
+  hs-source-dirs:      bench, test
+  main-is:             trace-dispatcher-bench.hs
+  default-language:    Haskell2010
+  default-extensions:  OverloadedStrings
+  other-modules:       Cardano.Logging.Test.Types
+                       Cardano.Logging.Test.Oracles
+                       Cardano.Logging.Test.Config
+                       Cardano.Logging.Test.Tracer
+                       Cardano.Logging.Test.Messages
+                       Cardano.Logging.Test.Script
+  build-depends:       base >=4.12 && <5
+                       , trace-dispatcher
+                       , criterion
+                       , containers
+                       , QuickCheck
+                       , text
+                       , aeson
+                       , time
+                       , ekg
```

Comment:
Maybe it would be better to arrange dependencies in alphabet order?

Reply:
I think it's our policy, actually.

Reply:
Done